CODEOWNERS

```diff
@@ -1 +1 @@
-* @MarshalX @MichalBor @MaorDavidzon @artem-fedorov @elsapet @gotbadger @cfabianski
+* @MarshalX @elsapet @gotbadger @cfabianski
```

Comment:
I think Phil needs to be added to the repo

Reply:
no! (kidding)